moto/core/models.py

```diff
@@ -12,11 +12,27 @@
 class MockAWS(object):
     nested_count = 0
 
-    def __init__(self, backends):
+    def __init__(self, backends, fixtures={}):
         self.backends = backends
 
         if self.__class__.nested_count == 0:
             HTTPretty.reset()
+        self.fixtures = fixtures
+
+    def inject_fixtures(self):
+        """
+        Create data in specified regions, example:
+
+            {'us-east-1': {'images': [{'ami_id': 'foo_id', 'name': 'foo_name'}]}}
+
+        Propreties are those used for instanciate a model instance.
+        """
+        from moto.ec2.models import Ami
+        for backend_name, backend in self.backends.items():
+            backend_fixtures = self.fixtures.get(backend_name, {})
+            for image in backend_fixtures.get('images', []):
```

Comment:
I'm a bit worried about how this code scales out when we start adding a lot more fixtures. Maybe we move this to the backends?